distributed/core.py

```diff
@@ -474,7 +474,7 @@ async def handle_comm(self, comm):
 
                 try:
                     op = msg.pop("op")
-                except KeyError as e:
+                except KeyError as e:  # pragma: no cover
```

Comment:
I have no idea what would actually happen if this exception is raised, i.e. what will the server do? Just continue working? Block? Close?
I think this should be tested

```suggestion
                except KeyError as e:
```